lib/cartopy/tests/mpl/test_mpl_integration.py

```diff
@@ -245,8 +245,13 @@ def test_cursor_values():
     MPL_VERSION.release[:2] < (3, 8), reason='mpl < 3.8')
 PARAMETRIZE_PCOLORMESH_WRAP = pytest.mark.parametrize(
     'as_rgba',
```

Comment:
Really minor thing, but I would change the 'as_rgba' variable name now that it’s no longer a boolean.  Maybe something like 'data_kind'?  I can always update it after merge though.

Reply:
Good idea. I went with "mesh_data_kind".